lusidtools/cocoon/utilities.py

```diff
@@ -19,6 +19,7 @@
 import time as default_time
 from lusidtools.cocoon.validator import Validator
 import types
+import uuid
```

Comment:
Where is this used in the code? It looks like you're generating your own uid from the timestamp hex?

Reply:
removed